.github/workflows/ci.yml

```diff
@@ -210,8 +210,53 @@ jobs:
       - name: Assert that git tree is clean
         run: git diff || echo "Run 'php build/scripts/generate-global-assert-wrappers.php' to regenerate global assert wrappers!"
 
-  build-and-test-phar:
-    name: Build and test PHAR
+  build-phar:
+    name: Build PHAR
+
+    runs-on: ubuntu-latest
+
+    env:
+      PHP_EXTENSIONS: none, dom, json, fileinfo, iconv, libxml, mbstring, pdo_sqlite, phar, soap, tokenizer, xml, xmlwriter
+      PHP_INI_VALUES: assert.exception=1, phar.readonly=0, zend.assertions=1
+
+    strategy:
+      matrix:
+        php-version:
+          - "8.1"
```

Comment:
I'm curious: why use matrix when there's only one option?

Reply:
@cesarkohl 

It's easier to manage and keeps the job in line with the other jobs - but of course, it is not necessary!

Reply:
Cool, I like the idea!